config.go

```diff
@@ -136,6 +136,13 @@ func (b *Builder) initConfig() {
 		b.OCIv1 = image
 		b.Docker = dimage
 	}
+	if len(b.Manifest) > 0 {
```

Comment:
The manifest is the top level of data in the image, and while the data type that it encodes differs depending on whether we're using docker or oci formats, it _should_ always be there.